web/src/engine/websites/ReadComicOnlineLI.ts

```diff
@@ -0,0 +1,40 @@
+import { Tags } from '../Tags';
+import icon from './ReadComicOnlineLI.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+function ChapterExtractor(anchor: HTMLAnchorElement) {
+    const link = new URL(anchor.href);
+    link.searchParams.set('readType', '1');
+    link.searchParams.set('quality', 'hq');
+    return {
+        id: link.pathname + link.search,
+        title: anchor.text.trim()
+    };
+}
+
+function MangInfoExtractor(element: HTMLDivElement) { //needed because titles are truncated in link
+    const anchor = element.querySelector<HTMLAnchorElement>('a');
+    const dom = new DOMParser().parseFromString(element.getAttribute('title'), 'text/html');
+    return {
+        id: anchor.pathname,
+        title: dom.querySelector('p.title').textContent.trim()
+    };
+}
+
+@Common.MangaCSS(/^{origin}\/Comic\/[^/]+$/, 'div.barContent a.bigChar')
+@Common.MangasMultiPageCSS('/ComicList?page={page}', 'div.list-comic div.item, div.item-list div.group div.col.info', 1, 1, 0, MangInfoExtractor)
+@Common.ChaptersSinglePageCSS('div.episodeList table.listing tr td:first-of-type a, div.section ul.list li a', ChapterExtractor)
+@Common.PagesSinglePageJS('lstImages', 2500) //may trigger a captcha request
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('readcomiconlineli', `ReadComicOnlineLI`, 'https://readcomiconline.li', Tags.Language.English, Tags.Media.Comic, Tags.Source.Aggregator);
```

Comment:
Consider to drop the `li` suffix since there is no other website with this name yet.

Reply:
I did that on purpose. We already got a ReadComicOnline.ru with such name.

![image](https://github.com/manga-download/haruneko/assets/10671518/939dadd4-2c86-4285-81cf-9b0b71888ed9)

And its a different website

Reply:
Yes the names are very similar, but the identifier `readcomiconline` is technically still different from `readcomicsonline`.
The `li` suffix may become confusing in case the website changes their top-level domain again.

Reply:
I shall remove the LI then 🤷